### ISSUE TITLE
Selected Tab in sideNav doesn't have sufficient contrast.

### ISSUE THREAD

**Issue by bhanupradeep7** [vmware/clarity#4552]
Ensure that interface controls have a luminosity contrast ratio of 3:1 or more.

Currently the background-color is #fafafa and selected tab background color is #d8e3e9. 
Current Contrast ratio is 1.25:1 .  This value has to be more than 3:1. 

Contract checker : https://www.levelaccess.com/color-contrast-checker/

![Screen Shot 2020-05-08 at 3 53 48 PM](https://user-images.githubusercontent.com/18656855/81397038-2d2dc780-9144-11ea-9a94-15eb2e6c7ce6.png)

**Comment by gnomeontherun** [comment]
Please use our bug report format when submitting issues, it helps us to properly understand your report. Otherwise we are less likely to triage and prioritize it without proper information.

From the limited information, it sounds like the report is the contrast between the two background colors is not considered sufficient. However, the text is also bolded to indicate selection. I don't believe this is an issue.

@chlane can you advise?

**Comment by chlane** [comment]
See https://github.com/vmware/clarity/issues/4378 as an example

**Comment by chlane** [comment]
This is a valid issue Nick @ncaskey04  and I checked it out.  The CSS causing this issue is below. Please choose a color combination that yields a 3:1 contrast ratio.

.sidenav .nav-group .nav-list .nav-link.active {
    background: #d8e3e9;
    background: var(--clr-sidenav-link-active-bg-color,#d8e3e9);
    color: #000;
    color: var(--clr-sidenav-link-active-color,#000);
}

.main-container {
    display: flex;
    flex-direction: column;
    height: 100vh;
    background: #fafafa;
    background: var(--clr-global-app-background,#fafafa);
}

**Comment by gnomeontherun** [comment]
There is a workaround available to update the color as needed for the immediate moment.

Clarity cares deeply about accessibility, and we work with our accessibility team to identify and triage issues. We have an internal tracking system to identify and verify accessibility issues, which has captured this already. Some issues with Clarity Angular are difficult to solve without causing side effects or breaking changes, and we have to balance concerns with fixing issues and breaking changes. As we build out Clarity Core, we are able to address these root cause accessibility issues and recommend adopting the Clarity Core implementations as they become available. We are closing this issue as we have the issues tracked internally. Please follow our development and releases to see when we release relevant Clarity Core components to make this possible.
